app/public/stylesheets/themes/minimal/style.css

```diff
@@ -0,0 +1,161 @@
+#pluginWrap {
+  display: flex;
+  height: 100vh;
+  display: flex;
+  align-items: center;
```

Comment:
This should be removed! With a count more than 24 it's slipping out of the window